deb/build/build.sh

```diff
@@ -13,7 +13,7 @@ mkdir -p $D
 cp -R $dir/* $D
 
 # Expand variables in the definition
-$BASE/bin/branding.sh $D/debian
+"$BASE/bin/branding.py" $D/debian
```

Comment:
If you mean switching from the shellscript to the python script, he's using this:  https://github.com/jenkinsci/packaging/blob/master/bin/branding.py

Something I whipped up as a safer way to apply branding (handles special characters safely, includes unit tests, and won't silently accept empty values).